<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Convolution</class>
 <widget class="QMainWindow" name="Convolution">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convolution</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="LayoutButtom">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="LayoutChart"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="LayoutData">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="start">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop">
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/new/icon/stop.png</normaloff>
              <disabledoff>:/new/icon/stopFalse.png</disabledoff>:/new/icon/stop.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resume">
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/new/icon/resume.png</normaloff>
              <disabledoff>:/new/icon/resumeFalse.png</disabledoff>
              <activeoff>:/new/icon/resume.png</activeoff>
              <selectedoff>:/new/icon/resume.png</selectedoff>:/new/icon/resume.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="help">
            <property name="minimumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="LayoutAfterData"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="LayoutBeforData"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
